validator/src/main.rs

```diff
@@ -2169,6 +2169,18 @@ pub fn main() {
                         .default_value(DEFAULT_RELAYER_MAX_FAILED_HEARTBEATS)
                 )
         )
+        .subcommand(
+            SubCommand::with_name("set-shred-receiver-address")
+                .about("Changes shred receiver address")
+                .arg(
+                    Arg::with_name("shred_receiver_address")
+                        .long("shred-receiver-address")
+                        .value_name("SHRED_RECEIVER_ADDRESS")
+                        .takes_value(true)
+                        .help("Validator will forward all shreds to this address in addition to normal turbine operation. Set to empty string to disable.")
+                        .required(true)
```

Comment:
must it be required?

Reply:
i think in the subcommand, we want it required,

see discussion here
https://github.com/jito-foundation/jito-solana/pull/304#discussion_r1197895880